tests/kernel/timer/timer_monotonic/src/main.c

```diff
@@ -33,10 +144,37 @@ int test_frequency(void)
 	return 0;
 }
 
+int test_frequency_msecond(void)
+{
+	u32_t start, end, delta, nanoseconds_spent, pct;
+
+	TC_PRINT("Testing system tick frequency\n");
+
+	start = k_cycle_get_32();
+	k_sleep(K_MSEC(1));
+	end = k_cycle_get_32();
+	delta = end - start;
+	nanoseconds_spent = (u32_t)k_cyc_to_ns_floor64(delta);
+	pct = (u64_t)delta * 100000U / sys_clock_hw_cycles_per_sec();
```

Comment:
Same note, this is really a "millipercent" fraction, might consider changing the name.